common_analysis_ip_and_uri_finder/ip_and_uri_finder_analysis.py

```diff
@@ -21,8 +20,8 @@ def get_strings_from_matches(matches):
 
     @staticmethod
     def get_file_content(file_path):
-        with open(file_path, "rb") as fp:
-            return fp.read()
+        with open(file_path, 'rb') as fd:
+            return fd.read()
```

Comment:
Can't we replace this with `Path.read_bytes()`? 

Maybe we can also "future proof" the module by adding optional support for Path objects as input (as in `def find_urls_in_file(self, file_path: str | Path):`) so that we can switch to Path objects in FACT.